test/e2e/clusterlocaldomaintls/cluster_local_domain_tls_test.go

```diff
@@ -46,8 +46,9 @@ func TestClusterLocalDomainTLSClusterLocalVisibility(t *testing.T) {
 		t.Skip("Alpha features not enabled")
 	}
 
-	if !(strings.Contains(test.ServingFlags.IngressClass, "kourier")) {
-		t.Skip("Skip this test for non-kourier ingress.")
+	if !strings.Contains(test.ServingFlags.IngressClass, "kourier") ||
+		strings.Contains(test.ServingFlags.IngressClass, "contour") {
+		t.Skip("Skip this test for non-kourier/contour ingress.")
```

Comment:
I am a bit confused, should not we remove skipping for contour?

Reply:
The check is inverted, so only run for kourier and now contour.

Reply:
actually the second clause isn't inverted

Reply:
Right, I missed that. Maybe use brackets around it then to make it more obvious?